pkg/tar/untar.go

```diff
@@ -0,0 +1,65 @@
+package tar
+
+import (
+	"archive/tar"
+	"io"
+	"os"
+)
+
+type UnPackager struct{}
```

Comment:
why don't we do one Package type with `func (Packager) Package` and `func (Packager) UnPackage`?